app/helpers/application_helper/button/instance_miq_request_new.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationHelper::Button::InstanceMiqRequestNew < ApplicationHelper::Button::Basic
+  def disabled?
+    !provisioning_supported?
+  end
+
+  def calculate_properties
+    super
+
+    if disabled?
+      self[:title] = N_('No Cloud Provider that supports instance provisioning added')
+    end
+  end
+
+  private
+
+  def provisioning_supported?
+    EmsCloud.all.any? { |provider| provider.supports_provisioning? }
```

Comment:
@romanblanco We've already had the discussion about whether this PR could be merged even though it's not a perfect solution. It works at least better. What do you think?